src/class/api.ga4gh.php

```diff
@@ -211,6 +214,11 @@ function __construct (&$oAPI)
         // To not duplicate code.
         $this->aValueMappings['effect'] = $_SETT['var_effect_api'];
 
+        // Retrieve default genome build.
+        global $_DB;
+        $this->aActiveGBs = $_DB->query('SELECT id, column_suffix FROM ' . TABLE_GENOME_BUILDS)->fetchAllCombine();
```

Comment:
I think that also here, we can fix a lot of suffix issues if we'd let the database concatenate the suffixes into the column names for us once, and we can just retrieve the required fields from this `SELECT` result.

Reply:
Yep you were very right; this has been done